ttter/accounts/views.py

```diff
@@ -10,39 +11,35 @@
 
 
 class UserDataInput(FormView):
-    template_name = 'accounts/create.html'
+    template_name = "accounts/create.html"
     form_class = SignUpFrom
 
     def form_valid(self, form):
-        return render(self.request, 'accounts/create.html',
-                      {'form': form})
+        return render(self.request, "accounts/create.html", {"form": form})
 
 
 class UserDataConfirm(FormView):
     form_class = SignUpFrom
 
     def form_valid(self, form):
-        return render(self.request, 'accounts/create_confirm.html',
-                      {'form': form})
+        return render(self.request, "accounts/create_confirm.html", {"form": form})
 
     def form_invalid(self, form):
-        return render(self.request, 'accounts/create.html',
-                      {'form': form})
+        return render(self.request, "accounts/create.html", {"form": form})
 
 
 class UserDataCreate(CreateView):
     form_class = SignUpFrom
-    success_url = reverse_lazy('base:top')
+    success_url = reverse_lazy("base:top")
 
     def from_valid(self, form):
```

Comment:
```suggestion
    def form_valid(self, form):
```
これ前回のレビュー漏れかもですが、form_validです。これに僕も気づかず延々とユーザー作成時にログイン状態にならなくてどうしてだろうと路頭に迷ってました笑

Reply:
自分のほうだと問題なくいってたの謎ですね

Reply:
いえ、うまく行ってないと思いますよ。もしうまく行っているなら、signupした後home画面にlogoutが表示されるはずです（signupしてもloginとsignupの文字があるはず）。from_validの状態でもユーザー自体は作成されますがログイン状態にはなっていなかったはずです。

テストでの確認方法として、ユーザー作成成功のtest（self.assertTrue(User.objects.exists()の下）に
`self.assertIn(SESSION_KEY, self.client.session)`を追記してみてください。from_validのままだと失敗するはずです。